code/modules/mob/living/living.dm

```diff
@@ -385,6 +385,7 @@
 
 /mob/living/proc/revive()
 	rejuvenate()
+	lying = 0
```

Comment:
No, just don't do hacky fix like this. You'll make people randomly stand up 

I'm quite sure that TG has an update_mobility proc that covers things properly but it'd be out of your scope to do it at the moment. So just remove this.